keyboards/keaboard/rules.mk

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
It might be best to simply remove this file.  Doing so would mean that you require the rev1 in the name to compile.